fdm-app/app/routes/farm.whats-new.tsx

```
@@ -0,0 +1,178 @@
+                                        {formatDistanceToNow(post.date, {
+                                            addSuffix: true,
+                                            locale: nl,
+                                        })}
+                                    </p>
+                                </div>
+                            </CardHeader>
+                            <Separator />
+                            <CardContent className="py-4">
+                                {/* Use ReactMarkdown to render the description */}
+                                <ReactMarkdown
+                                    children={post.description}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)